game_eggs/steamcmd_servers/neosvr/egg-neos-v-r.json

```diff
@@ -0,0 +1,82 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-06-03T10:59:04-04:00",
+    "name": "NeosVR",
+    "author": "espeon@espeon.dev",
+    "description": "Neos VR is a metaverse engineered for the unknown! It\u2019s one of the most versatile and feature rich metaverses for virtual reality, designed so everybody can find something interesting or useful to do in a social setting.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:mono_latest": "ghcr.io\/parkervcp\/yolks:mono_latest"
+    },
+    "file_denylist": [],
+    "startup": "exec mono Neos.exe -c .\/Config\/Config.json -l .\/Logs $(if {{ENABLE_MODLOADER}}; then echo \"-LoadAssembly .\/NeosModLoaderHeadless.dll\"; fi)",
+    "config": {
+        "files": "{}",
```

Comment:
please force a ports as this can give issue with other servers

Reply:
The game's networking uses some weird NAT traversal library + their own custom relay servers, so no ports need to be set by default as the entire game is meant to never need to forward ports. The library is called: https://github.com/RevenantX/LiteNetLib

Reply:
it is fine, I discused this with Jolty in the discord